test/fixtures/invalid-partials/duplicate-keys.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Let's make these files `.erb`--theoretically the render task could be optimized to treat plain yamls differently (the renderer itself is IIRC)

Suggestion: make this fixture reflect the use case in the PR description: the duplicate keys being generated by an ERB loop without the appropriate separator placement